src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,11 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Ball firstBall = new Lottery().getRandomBall();
```

Comment:
It's more efficient to create one instance of the Lottery class and then call the getRandomBall() method multiple times. 🚀